.github/container/manifest.yaml

```diff
@@ -132,12 +132,6 @@ mujoco:
   tracking_ref: main
   latest_verified_commit: ced37ffbb237584512311b041bce3124e3b2cc2a
   mode: git-clone
-grain:
-  # Used only in ARM t5x builds
-  url: https://github.com/google/grain.git
-  tracking_ref: main
-  latest_verified_commit: 41f80fb540547ec3aefe7cec57fe43228560b1cf
-  mode: git-clone
```

Comment:
Could we add something like:
checkout: hash

to be more consistent and simplify it when this case happens?

Reply:
I guess this would be analogous to the current `mode: git-clone`, but pinning the latest verified commit?

(Though I would still merge this now to unblock)

Reply:
Missed today's nightly.
I think it would be cleaner if we could just add one line of YAML, something like
```yaml
grain:
  pin: true
```
that would tell `bump.sh` not to advance the commit hash, and thereby make the nightlies use the pinned commit.
That said, that feature should be used sparingly and always temporarily.

Reply:
Does setting `tracking_ref: 41f80fb540547ec3aefe7cec57fe43228560b1cf` work?